src/styles/index.ts

```diff
@@ -2889,6 +2891,11 @@ const styles = (theme: ThemeColors) =>
             alignItems: 'center',
         },
 
+        radioButtonLockIcon: {
```

Comment:
I think a better name is toggleSwitchLockIcon, since this UI component is a toggle and not a radio button

Reply:
But that being said, we already have this saved as a variable for `iconSizeExtraSmall`:

![CleanShot 2024-04-09 at 09 30 23@2x](https://github.com/Expensify/App/assets/2319350/1ae4fdb6-4dea-4293-8972-9dfeeb54d356)